packages/scene-composer/src/components/three-fiber/EditorTransformControls.tsx

```diff
@@ -59,7 +58,7 @@ export function EditorTransformControls() {
 
   useEffect(() => {
     if (selectedSceneNode) {
-      if (isTagComponent && !tagResizeEnabled && transformControlMode === 'scale') {
+      if (isTagComponent && transformControlMode === 'scale') {
```

Comment:
Is there a reason you hardcoded the value `scale` and `translate`?

Reply:
the type `TransformControlMode` is defined as 
```
export type TransformControlMode = 'translate' | 'rotate' | 'scale';
```
so comparing it with strings other than those three will show type error